src/Extensions/ComposerLoaderExtension.php

```diff
@@ -130,7 +118,7 @@ public function onAfterBuild()
         $this->setComposer($composer);
 
         if ($originalDir !== BASE_PATH) {
-            chdir($originalDir);
+            chdir($originalDir ?? '');
```

Comment:
`$originalDir` is either a string or false, never null. 
```suggestion
            chdir($originalDir ?: '');
```

Reply:
Reverted